pkg/kubectl/cmd/get.go

```diff
@@ -315,5 +339,5 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			continue
 		}
 	}
-	return utilerrors.Flatten(utilerrors.NewAggregate(allErrs))
+	return utilerrors.NewAggregate(allErrs)
```

Comment:
The split you're referring to of using Visit and non-Visit was for the `generic` output case - I've since changed that in my most recent push so that they both make use of `Infos()`.

If I'm understanding you correctly, you're referring to the block of code _after_ the generic case, and how that should be altered to use Visit() to handle the printing correctly. Did I get that right?